content/arduino-cloud/09.business/02.aws-s3-exporter/content.md

```diff
@@ -0,0 +1,465 @@
+* Use CloudFormation templates to simplify deployment and configuration.  
+* Learn how Lambda, CloudWatch and EventBridge help monitor the deployed CloudFormation stack.  
+
+## Required Software
+
+* [Arduino Cloud](https://cloud.arduino.cc/). **If you do not have an account, you can create one for free inside [cloud.arduino.cc](https://cloud.arduino.cc/home/?get-started=true)**.
+* [AWS CLI](https://aws.amazon.com/cli/). **If you do not have an AWS account, please refer to the [online AWS documentation](https://docs.aws.amazon.com/iot/latest/developerguide/setting-up.html)**.
+* [Go Programming Language](https://go.dev/) (version 1.22 or higher) (Optional)
+* [Official Arduino AWS S3 CSV Exporter Repository](https://github.com/arduino/aws-s3-integration)
+
+## How It Works
+
+The **Arduino AWS S3 CSV Exporter** extracts time series data from **Arduino Cloud** and publishes it to an **AWS S3** bucket. Data extraction is handled by an AWS Lambda function written in **GO**, which runs at scheduled intervals using **AWS EventBridge**.
+
+Each function execution retrieves data from the selected **Arduino Things** and generates a CSV file. The file is then uploaded to **S3** for structured storage and accessibility.
+
+Data is extracted every hour by default, with samples aggregated at a 5 minute resolution. Both the extraction period and the aggregation rate are configurable. Aggregation is performed by calculating the average over the aggregation period, while non-numeric values, such as strings, are sampled at the specified resolution.
+
+Time-series data is exported in **UTC** by default. All Arduino Things in the account are included in the export unless filtered using [**tags**](#tag-filtering). 
+
+This setup provides a structured and scalable approach for managing time series data from connected devices, providing configurable parameters such as sampling intervals and data filtering.
+
+## AWS Account & CloudFormation Template
+
+An active AWS account is required to deploy the **Arduino AWS S3 CSV Exporter**. If an account is not available, refer to the [online AWS documentation](https://docs.aws.amazon.com/iot/latest/developerguide/setting-up.html) for account setup. The following steps can help you to get started:
+
+- [Sign up for an AWS account](https://docs.aws.amazon.com/iot/latest/developerguide/setting-up.html#aws-registration)
+- [Create an administrative user](https://docs.aws.amazon.com/iot/latest/developerguide/setting-up.html#create-an-admin)
+
+The exporter setup involves deploying resources using a [**CloudFormation template**](https://github.com/arduino/aws-s3-integration/blob/0.3.0/deployment/cloud-formation-template/deployment.yaml). This template provisions and configures the necessary AWS resources automatically.
+
+CloudFormation requires the following **IAM permissions** to automatically provision and manage the AWS resources used in this deployment.
```

Comment:
What is IAM? I suggest adding a reference to these AWS terms.

Reply:
```suggestion
AWS CloudFormation requires specific [**IAM permissions**](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) to automatically provision and manage AWS resources during this deployment. AWS Identity and Access Management (IAM) is a web service that helps you securely control access to AWS resources by managing permissions and determining which resources users can access.
```

Reply:
Applying this change! thanks